package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "0.5.7",
+  "version": "0.5.8",
```

Comment:
You also need to update (and commit) the package-lock file.

Reply:
ok. Done!

Reply:
@andrepestana-aot Let the person who created a conversation resolve it -- in my case it's my checklist to verify that the thing I commented on is fixed.

If you want to indicate that you're read or handled this thing, you can add a 👍  to the comment.

Reply:
ok